contrib/kubespray/example/config.yml

```diff
@@ -1,7 +1,7 @@
 user: forexample
 password: forexample
 branch-name: master
-docker-image-tag: quick-start
```

Comment:
Can we change it to latest release version? Or use `<% latest-release %>` to refer the latest version.

Reply:
Done